integrations/tensorflow-keras/Neptune-TensorFlow-Keras.ipynb

```diff
@@ -0,0 +1,681 @@
+    "\n",
+    "if set(all_logs.keys()) != set(correct_logs):\n",
+    "    raise ValueError('incorrect metrics')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "YHQrdIgOIo_E",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## More Options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "You can log way more than just metrics to Neptune. \n",
+    "\n",
+    "Let's go over some of those options here. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Q9zxE-EzLAbe",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Log hardware consumption"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "To log GPU/CPU and memory during training you just need to have ``psutil`` installed and run your logger as you normally would. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet psutil==5.6.6"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "installation",
+     "library_updates"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet psutil"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iAJZLVQKL8Dt",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Log hyperparameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "If you want to log hyperparameters of your model training run you just need to pass a parameters dictionary into the ``.create_experiment`` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Hw_lQq9AL5PR",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "PARAMS = {'lr':0.005, \n",
+    "          'momentum':0.9, \n",
+    "          'epochs':10,\n",
+    "          'batch_size':32}\n",
+    "\n",
+    "optimizer = tf.keras.optimizers.SGD(lr=PARAMS['lr'], momentum=PARAMS['momentum'])\n",
+    "\n",
+    "model.compile(optimizer=optimizer,\n",
+    "              loss='sparse_categorical_crossentropy',\n",
+    "              metrics=['accuracy'])\n",
+    "\n",
+    "# log params\n",
+    "neptune.create_experiment('tensorflow-keras-advanced', params=PARAMS)\n",
+    "\n",
+    "model.fit(x_train, y_train,\n",
+    "          epochs=PARAMS['epochs'],\n",
+    "          batch_size=PARAMS['batch_size'],\n",
+    "          callbacks=[NeptuneMonitor()])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Log image predictions"
```

Comment:
Missing comment above code - you did it for all above, so maybe you want to be consistent.
You can also advertise log image method.